features/src/jupyter/install.sh

```diff
@@ -0,0 +1,991 @@
+#!/usr/bin/env bash
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
removethe copyright

Reply:
Is that usually how it works when we make modifications? Idk how code licensing works